docs/src/main/paradox/user/lang/java/eventsourced.md

```diff
@@ -6,15 +6,15 @@ An event sourced entity can be created by annotating it with the @javadoc[`@Even
 
 @@snip [ShoppingCartEntity.java](/docs/src/test/java/docs/user/eventsourced/ShoppingCartEntity.java) { #entity-class }
 
-The `persistenceId` is used to namespace events in the journal, useful for when you share the same database between multiple entities. It defaults to the simple name for the entity class (in this case, `ShoppingCartEntity`), it's good practice to select one explicitly, this means your database isn't depend on classnames in your code.
+The `persistenceId` is used to namespace events in the journal, useful when you share the same database between multiple entities. It defaults to the simple name for the entity class (in this case, `ShoppingCartEntity`). It is a good practice to define an explicit persistenceId. This helps to decouple the journal data from the classnames in your code. ie. the events journal remain coherent, even if the underlying entity classname may have changed.
 
-The `snapshotEvery` parameter controls how often snapshots are taken, so that the entity doesn't need to be recovered from the whole journal each time it's loaded. If left unset, it defaults to 100. Setting it to a negative number will result in snapshots never being taken.
+The `snapshotEvery` parameter controls how often snapshots are taken, so that the entity doesn't need to be recovered from the entire history of the journal each time it is loaded. If left unset, it defaults to 100. To disable snapshot, set `snapshotEvery` to a negative value.
```

Comment:
```suggestion
The `snapshotEvery` parameter controls how often snapshots are taken, so that the entity doesn't need to be recovered from the entire history of the journal each time it is loaded. If left unset, it defaults to 100. To disable snapshots, set `snapshotEvery` to a negative value.
```